common/server_room.cpp

```diff
@@ -232,6 +234,22 @@ void Server_Room::say(const QString &userName, const QString &s, bool sendToIsl)
     event.set_name(userName.toStdString());
     event.set_message(s.toStdString());
     sendRoomEvent(prepareRoomEvent(event), sendToIsl);
+ 
+    if (chatHistorySize != 0) {
+
+        ServerInfo_ChatMessage chatMessage;
+        QDateTime dateTime = dateTime.currentDateTime();
+        QString dateTimeString = dateTime.toString("MM/dd/yy hh:mm:ss");
```

Comment:
Can i suggest Qt::ISODate ? mm/dd/yy is used only in the USA (https://en.wikipedia.org/wiki/Date_format_by_country)